_posts/2020-06-20-snap-minis.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Snap Minis: How we can help you bring your app into SnapChat"
+date: 2020-06-20 11:00:00
+reviewed: 2020-06-20 11:00:00
+categories: []
+author: ABizzinotto
+---
+
+Last week, [Snap](https://www.snap.com) announced a new product during the [Snap Partner Summit 2020](https://www.youtube.com/watch?v=BfvubHha69k), [Snap Minis](https://minis.snapchat.com/). Minis are bite-sized third-party applications that live inside [Snapchat](https://www.snapchat.com/) and can be accessed through Snapchat's Chat interface. They are built on the same principle and technology as the Snap Games and are easy to develop, allowing you to have an app that takes advantage of the existing [Snapchat](https://www.snapchat.com/) application and is available to Snapchat users worldwide.
```

Comment:
This is not that important, but personally I don't like relative times (Last week) since as a user you then have to search for the article's date. And even if the date is like... 100px above (it's small and with not that great contrast so someone may miss that, looks even smaller on mobile). Maybe we can use `On june 2020....`? It's not that important though if you want to stay with `Last week`.

Reply:
Later on, we'll need to update this to a relative date. But right now, I'm using "last week" to highlight how "new" this product is and the fact that we're already working with it :)